Bio/Seq.py

```diff
@@ -2310,7 +2314,7 @@ def transcribe(dna):
     elif isinstance(dna, MutableSeq):
         return dna.toseq().transcribe()
     else:
-        return dna.replace("T", "U").replace("t", "u")
+        return str.translate(dna, {84: 85, 116: 117})  # str.maketrans("Tt", "Uu")
```

Comment:
Using `str.translate` is about 10% faster than using `.replace`. But this is for plain strings also, which doesn't need to be fast, as Biopython internally uses `Seq` objects anyway.

Reply:
So this change might be worthwhile on this else-branch which is for string input?

Reply:
Personally I wouldn't do this. It only matters for long strings, and I don't know where they could be coming from in Biopython. Unless somebody wants to type in a very very long string, you won't notics any speed improvements in practice.